pycircuitbreaker/pycircuitbreaker.py

```diff
@@ -160,13 +160,32 @@ def success_count(self) -> int:
         return self._strategy.success_count
 
 
+class CircuitBreakerRegistry:
+    def __init__(self) -> None:
+        self._registry: List[CircuitBreaker] = []
+
+    def register(self, circuitbreaker: CircuitBreaker) -> None:
+        found = [
```

Comment:
With a lot of breakers this would start to slow down. Would a dict of `id: CircuitBreaker` make more sense? Another option is to keep a second set of all the IDs so that the check is quick